bears/general/SpaceConsistencyBear.py

```diff
@@ -38,7 +41,47 @@ def run(self,
         result_texts = []
         additional_info_texts = []
 
-        for line_number, line in enumerate(file, start=1):
+        def get_blanklines_nr_start():
+            line_nr_start = False
+            rev_enumerated_zip_obj = zip(reversed(range(1, len(file) + 1)),
+                                         reversed(file))
+            for line_number, line in rev_enumerated_zip_obj:
+                replacement = line
+                if replacement.strip() == '':
+                    line_nr_start = line_number
+                else:
+                    break
+
+            return line_nr_start
```

Comment:
1. You don't need this code to reside inside a function as you only use it once.
2. This can be simplified to a single expression:
    ```
    line_nr_start = next(i for i, line in reversed(list(enumerate(file))) if line.strip() != '')
    ```